website/source/docs/providers/aws/d/db_snapshot.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "aws"
+page_title: "AWS: aws_db_snapshot"
+sidebar_current: "docs-aws-datasource-db-snapshot"
+description: |-
+  Get information on a DB Snapshot.
+---
+
+# aws\_db\_snapshot
```

Comment:
@catsby this follows the same pattern as the other data sources - it can indeed return multiple results but we try to enforce people to return just 1.... for now